### ISSUE TITLE
Harmonize the description of several standard names for various types of precipitation

### ISSUE THREAD

**Issue by larsbarring** [cf-convention/discuss#262]
Some time ago, when looking at #159, I discovered inconsistent and/or rather too brief description(s) of several standard names for various types precipitation. Hence I think that it would be useful to make a concerted effort to harmonise these. 

problem | standard name | description | canonical unit
-- | -- | -- | --
too brief| convective_rainfall_amount | "Amount" means mass per unit area. | kg m-2
too brief | convective_rainfall_flux | In accordance with common usage in geophysical disciplines, "flux" implies per unit area, called "flux density" in physics. | kg m-2 s-1
too brief | convective_rainfall_rate | _No help available_ | m s-1
too brief | rainfall_amount | "Amount" means mass per unit area. | kg m-2
too brief | rainfall_flux | In accordance with common usage in geophysical disciplines, "flux" implies per unit area, called "flux density" in physics. | kg m-2 s-1
too brief | rainfall_rate | _No help available_ | m s-1
term mixup | stratiform_`graupel`_ flux Alias: large_scale_`graupel`_ flux | In accordance with common usage in geophysical disciplines, "flux" implies per unit area, called "flux density" in physics. Stratiform `precipitation, whether liquid or frozen, is precipitation` that formed in stratiform cloud. Graupel consists of heavily rimed snow particles, often called snow pellets; often indistinguishable from very small soft hail except when the size convention that hail must have a diameter greater than 5 mm is adopted. Reference: American Meteorological Society Glossary http://glossary.ametsoc.org/wiki/Graupel. There are also separate standard names for hail. Standard names for "graupel_and_hail" should be used to describe data produced by models that do not distinguish between hail and graupel. | kg m-2 s-1
term mixup (#159) | stratiform_`rainfall`_amount Alias: large_scale_`rainfall`_amount | Stratiform `precipitation`, whether liquid or frozen, is `precipitation` that formed in stratiform cloud. "Amount" means mass per unit area. | kg m-2
term mixup | stratiform_`rainfall`_flux Alias: large_scale_`rainfall`_flux | In accordance with common usage in geophysical disciplines, "flux" implies per unit area, called "flux density" in physics. Stratiform `precipitation`, whether liquid or frozen, is `precipitation` that formed in stratiform cloud. | kg m-2 s-1
term mixup | stratiform_`rainfall`_rate Alias: large_scale_`rainfall`_rate | Stratiform `precipitation`, whether liquid or frozen, is `precipitation` that formed in stratiform cloud. | m s-1
term mixup| stratiform_`snowfall`_amount Alias: large_scale_`snowfall`_amount | Stratiform `precipitation`, whether liquid or frozen, is `precipitation` that formed in stratiform cloud. "Amount" means mass per unit area. | kg m-2
term mixup | stratiform_`snowfall`_flux Alias: large_scale_`snowfall`_flux | In accordance with common usage in geophysical disciplines, "flux" implies per unit area, called "flux density" in physics. Stratiform `precipitation`, whether liquid or frozen, is `precipitation` that formed in stratiform cloud. | kg m-2 s-1

This table only include standard names where I think that there is room for improvements and/or clarifications. Related to these 12 ones there are 43 other standard names from which useful boilerplate pieces of text can be extracted, or otherwise used as inspiration. Thus I believe the work involved is not overwhelming to improve the description of these 12 standard names.

In addition to these standard names in need of better descriptions, there is one standard name that I believe is wrong or at least inconsistent:

standard name | description | canonical unit
-- | --| --
convective_precipitation_rate | "Precipitation rate" means the depth or thickness of the layer formed by precipitation per unit time. Convective precipitation is that produced by the convection schemes in an atmosphere model. "Precipitation" in the earth's atmosphere means precipitation of water in all phases. | m s-1 | convective_precipitation_rate is the sum of convective_rainfall_rate and what (and the three non-existing convective_snow_\|  _hail_ \| _graupel_fall_rate)? This would be inconsistent because of what the height represents. This quantity is better represented by the already existing lwe_convective_precipitation_rate

The problem with this is that **convective_`precipitation`_rate** (supposedly, and in line with other standard names) is the sum of **convective_`rainfall`_rate** and _what_ (and the three non-existing **convective_`snow`_|  _`hail`_ | _`graupel`_fall_rate**)? 

This would be inconsistent because of what the height represents. This quantity is better represented by the already existing **lwe_convective_precipitation_rate**. Thus I suggest that **convective_`precipitation`_rate** is deprecated in favour of **lwe_convective_precipitation_rate**.

**Comment by larsbarring** [comment]
ping @japamment for inclusion as a hackaton theme in the schedule for the CF2023 workshop.

**Comment by efisher008** [comment]
Hello @ethanrd, @japamment,

Was there more offline discussion on this topic in the 2023 CF workshop in October?

Best wishes,
Ellie

**Comment by larsbarring** [comment]
Yes, thanks for reminding me. I will update with some notes from our breakout conversation.

**Comment by efisher008** [comment]
Hello @larsbarring,

> Yes, thanks for reminding me. I will update with some notes from our breakout conversation.

Would you be able to update the post with notes from the conversation at the CF workshop, please? It looks like the last edit to the original post was October and it would be useful to have any more discussion recorded. Thank you! 

Best wishes,
Ellie
